src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -910,6 +879,7 @@ def check_connection_and_deployment(flow_folder, connection, deployment):
                 assert not (flow_folder / "azure_openai.yaml").exists()
                 assert not (flow_folder / "openai.yaml").exists()
 
+    @pytest.mark.skipif(RecordStorage.is_replaying_mode(), reason="cannot support interactive")
```

Comment:
btw, why cannot support interactive?

Reply:
It returns a stream object. Shelve record this and I'm not sure if I have
Skip for now and I'll investigate further